tests/product/base_product_case.py

```diff
@@ -108,69 +106,55 @@ class BaseProductTestCase(BaseTestCase):
     def setUp(self):
         super(BaseProductTestCase, self).setUp()
         self.maxDiff = None
-        self.presto_rpm_filename = self.detect_presto_rpm()
         self.cluster = None
+        self.default_keywords = {}
+
+    def _run_installers(self, installers):
+        cluster = self.cluster
+        for installer in installers:
+            dependencies = installer.get_dependencies()
+
+            for dependency in dependencies:
+                dependency.assert_installed(self, cluster.master)
```

Comment:
How about if cluster exposes a get_master() method for installers and then they can pick?